androidapp/app/src/main/java/com/example/lanweather/ui/settings/SettingsFragment.kt

```diff
@@ -27,7 +27,7 @@ class SettingsFragment : PreferenceFragmentCompat() {
 //
 //    override fun onActivityCreated(savedInstanceState: Bundle?) {
 //        super.onActivityCreated(savedInstanceState)
-//        viewModel = ViewModelProviders.of(this).get(SettingsViewModel::class.java)
+//        viewModel = ViewModelProvider(this).get(SettingsViewModel::class.java)
```

Comment:
more function changes in comments? i'm curious what caused these

Reply:
In this case, I changed it to an updated convention in case we ever decide to properly use the ViewModelProvider. I believe ViewModelProviders.of() is deprecated.